src/model/paservice/base_data_accessor_pa_service.py

```diff
@@ -185,4 +191,5 @@ def build_request(external_id: str, filter: dict[str, Any]) -> dict[str, Any]:
     ]
     if weights:
         request_body["weights"] = weights
+    print(request_body)
```

Comment:
Die Debug Ausgabe kann wohl raus. Oder in logging umwandeln wenn nötig / gewünscht

Reply:
vollkommen richtig, ist erledigt